### ISSUE TITLE
Not an issue but a compliment

### ISSUE THREAD

**Issue by polymathuk** [6cdh/SimpleMermaid#1]
This is honestly brilliant. 15 seconds to install and enable and it works out of the box. Thanks.

**Comment by 6cdh** [comment]
Yeah. It would be easier to get JS things work on client rather than server :)
